portfolio.css

```diff
@@ -0,0 +1,245 @@
+*{
+	box-sizing: border-box;
+	color:#fff;
+}
+
+/* Background image css */
+
+body{
+	background-image: url(mountain_bg.jpg);
+	background-repeat: no-repeat;
+	background-position: bottom;
+	background-color:#FBEEC1;
+	background-attachment:fixed;
+	background-size:cover;
+}
+
+/* Header formatting */
+
+h1, h2{
+	background-color:#115464;
+	border-radius:16px;
+	border-style: solid;
```

Comment:
Can combine border color, style and width into 1 line. Use border: width style color;   :)